### ISSUE TITLE
Mock tests should be independent

### ISSUE THREAD

**Issue by sosnovsky** [FlowCrypt/flowcrypt-browser#5835]
Thank you for your hint. I thought tests were isolated and tests each other don't affect others. But seems like add/generate key affects other tests

_Originally posted by @ioanmo226 in https://github.com/FlowCrypt/flowcrypt-browser/issues/5823#issuecomment-2346241125_

**Comment by sosnovsky** [comment]
Yes, tests should be isolated and don't affect each other, as they're expected to have separate mock environments.
Let's check why adding key affected other tests and fix it.

**Comment by ioanmo226** [comment]
During my review of the `settings - my key page - generate key` test failure, I discovered that other tests were failing because I initially attempted to save the backup to the Inbox. After switching to `no-backup` or `backup to file`, the tests passed.

It seems like the Inbox is shared across all tests. I'll investigate why this is happening.

**Comment by sosnovsky** [comment]
Thanks for checking, let's leave it for now, I'll move it to `Later` milestone
